lib/logstash/config/grammar.rb

```diff
@@ -1725,20 +1724,33 @@ def _nt_hashentries
   end
 
   module Hashentry0
-    def name
+    def _
```

Comment:
who what .. why! this code. 

Is this generated by a newer version of treetop? It's crazysauce. :P